### ISSUE TITLE
Should I switch to "checkpint sync" now?

### ISSUE THREAD

**Issue by kevgithub** [ethereum/go-ethereum#28044]
My node has been syncing for a while now(8 days), and here's the latest log:
CL:
```
time="2023-09-01 14:54:37" level=info msg="Called new payload with optimistic block" payloadBlockHash=0xe8c18e0af93a prefix=blockchain slot=6232378
time="2023-09-01 14:54:37" level=info msg="Called new payload with optimistic block" payloadBlockHash=0x26dc03d219f1 prefix=blockchain slot=6232379
time="2023-09-01 14:54:37" level=info msg="Called new payload with optimistic block" payloadBlockHash=0x72f1ddbe714e prefix=blockchain slot=6232380
time="2023-09-01 14:54:37" level=info msg="Called new payload with optimistic block" payloadBlockHash=0x2b8d9858bb44 prefix=blockchain slot=6232381
time="2023-09-01 14:54:37" level=info msg="Called new payload with optimistic block" payloadBlockHash=0x768649bc5352 prefix=blockchain slot=6232382
time="2023-09-01 14:54:37" level=info msg="Called new payload with optimistic block" payloadBlockHash=0x6c5d773eec08 prefix=blockchain slot=6232383
time="2023-09-01 14:54:37" level=info msg="Called fork choice updated with optimistic block" finalizedPayloadBlockHash=0x9765c1402bd5 headPayloadBlockHash=0x6c5d773eec08 headSlot=6232383 prefix=blockchain
time="2023-09-01 14:54:37" level=info msg="Processing block batch of size 58 starting from  0x7334f1cd... 6232384/7227271 - estimated time remaining 30h2m19s" blocksPerSecond=9.2 peers=35 prefix=initial-sync
```

EL:
```
INFO [09-01|14:55:37.676] Syncing: chain download in progress      synced=100.00% chain=327.56GiB headers=17,056,775@6.60GiB bodies=17,056,677@215.96GiB receipts=17,056,677@105.00GiB eta=29.202s
WARN [09-01|14:55:37.945] Pivot seemingly stale, moving            old=17,056,678 new=17,056,741
INFO [09-01|14:55:38.745] Syncing: state healing in progress       accounts=0@0.00B slots=0@0.00B codes=34716@259.11MiB nodes=97,790,751@7.55GiB pending=1
INFO [09-01|14:55:44.472] Forkchoice requested sync to new head    number=17,056,869 hash=13e9c5..e116e4 finalized=17,056,806
INFO [09-01|14:55:45.750] Syncing: chain download in progress      synced=100.00% chain=327.56GiB headers=17,056,838@6.60GiB bodies=17,056,740@215.96GiB receipts=17,056,740@105.00GiB eta=29.395s
WARN [09-01|14:55:46.948] Pivot seemingly stale, moving            old=17,056,741 new=17,056,805
INFO [09-01|14:55:47.819] Syncing: state healing in progress       accounts=0@0.00B slots=0@0.00B codes=34716@259.11MiB nodes=97,790,751@7.55GiB pending=1
```
eth.syncing:
```
{
  currentBlock: 17057747,
  healedBytecodeBytes: 271694812,
  healedBytecodes: 34716,
  healedTrienodeBytes: 8106227747,
  healedTrienodes: 97790751,
  healingBytecode: 0,
  healingTrienodes: 0,
  highestBlock: 17057812,
  startingBlock: 17033083,
  syncedAccountBytes: 47670286424,
  syncedAccounts: 192520528,
  syncedBytecodeBytes: 5057491353,
  syncedBytecodes: 763178,
  syncedStorage: 831341321,
  syncedStorageBytes: 178645236259
}
```
 Now I'm considering about whether I should switch to the checkpoint method of syncing, and I have two questions: 

1, if I switch to the checkpoint method now, do I need to clear the data and start syncing over? 
2, I'm not sure how much longer the current sync method will take, if I switch to checkpoint now, will I be able to finish the sync in a shorter period of time?
 Thanks a lot!

**Comment by Deacon77** [comment]
I'm not sure what you're saying but I'm not going to lie to you about this because you don't want it to happen and I'm going to have it happen

**Comment by namiloh** [comment]
@kevgithub that looks very strange. Your state-heal, after 9 days, should have finished long ago. What are your commmand parameters, and are you using an ssd or hdd backend?

Alas there's no checkpoint-sync method to switch to, for EL. You are already using the fastest way to sync (usually takes a day or two)

**Comment by MariusVanDerWijden** [comment]
Yes you should move to checkpoint sync on the consensus layer.
Right now the CL node feeds us old blocks one by one, so we can't finish our healing (because we try to fetch old states from peers that they don't have anymore)

**Comment by Deacon77** [comment]
﻿﻿On Sep 5, 2023, at 2:06 AM, Marius van der ﻿ ***@***.***>﻿
Closed #28044 as completed.

**Comment by Deacon77** [comment]
﻿﻿﻿ ﻿﻿On Sep 5, 2023, at 2:06 AM, Marius van der Wijden ***@***.***> wrote:﻿
Closed #28044 as completed.

**Comment by Deacon77** [comment]
﻿﻿﻿﻿﻿﻿cc﻿ ﻿﻿On Sep 5, 2023, at 2:06 AM, Marius van der Wijden ***@***.***> wrote:﻿
Closed #28044 as completed.

**Comment by Deacon77** [comment]
﻿﻿﻿﻿﻿﻿﻿cc﻿ ﻿﻿On Sep 5, 2023, at 2:06 AM, Marius van der Wijden ***@***.***> wrote:﻿
Closed #28044 as completed.
